### PR TITLE
add documentation

### PR DESCRIPTION
**Pull request by ltalirz**
Start fixing https://github.com/lsmo-epfl/aiida-lsmo/issues/18

 * add basic documentation template for RTD
 * fix formatting of some docstrings
 * fix issue that broke RTD build

### CODE REVIEW

**Comment by ltalirz** [comment]
@yakutovicha @danieleongari 
It was what I suspected but it was not easy to find ...
We definitely need to find a way to get proper error messages for this

**Comment by ltalirz** [comment]
@yakutovicha you can try building the documentation locally and take it from here

for travis setup, see e.g. aiida-lsmo

File: aiida_raspa/utils/base_parser.py [hunk 1]
```diff
@@ -41,16 +41,20 @@ def float(number):  # pylint: disable=redefined-builtin
...
-        Block[ 0]        12025.61229 [A^3]
-        Block[ 1]        12025.61229 [A^3]
-        Block[ 2]        12025.61229 [A^3]
-        Block[ 3]        12025.61229 [A^3]
-        Block[ 4]        12025.61229 [A^3]
-        ------------------------------------------------------------------------------
-        Average          12025.61229 [A^3] +/-            0.00000 [A^3]
+    """Parse volume block.
+
+    Parse block that looks as follows::
```
Comment (yakutovicha) [inline aiida_raspa/utils/base_parser.py]: Here it is not only about the volume block. There are many other blocks that have the same structure.

**Comment by yakutovicha** [comment]
@ltalirz can you please also add the doc build tests to this PR? Once this is done, I am happy to merge it.

**Comment by ltalirz** [comment]
I've added the build of the docs on travis.

Do you have an idea why travis no longer runs?

**Comment by yakutovicha** [comment]
Btw, it's failed: https://travis-ci.org/yakutovicha/aiida-raspa

**Comment by yakutovicha** [comment]
Travis should work now, I believe. Please try to push something.

**Comment by yakutovicha** [comment]
oh, Travis is alive again.

**Comment by ltalirz** [comment]
tests pass

File: README.md [hunk 2]
```diff
@@ -212,4 +212,4 @@ This work is supported by:
 
 <img src="miscellaneous/logos/MARVEL.png" alt="MARVEL" style="padding:10px;" width="150"/>
 <img src="miscellaneous/logos/MaX.png" alt="MaX" style="padding:10px;" width="250"/>
-<img src="miscellaneous/logos/swissuniversities.png" alt="swissuniversities" style="padding:10px;" width="250"/>
\ No newline at end of file
+<img src="miscellaneous/logos/swissuniversities.png" alt="swissuniversities" style="padding:10px;" width="250"/>
```
Comment (yakutovicha) [inline README.md]: I wonder: what is the difference here.

**Comment by yakutovicha** [comment]
muchas gracias @ltalirz!

**Comment by yakutovicha** [comment]
something didn't work on reathedocs, any ideas @ltalirz ?

```
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
processing aiida-raspa.tex... index workflows apidoc/aiida_raspa apidoc/aiida_raspa.calculations apidoc/aiida_raspa.parsers apidoc/aiida_raspa.utils apidoc/aiida_raspa.workchains 
resolving references...
done
writing... failed

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/aiida-raspa/envs/develop/lib/python3.6/site-packages/sphinx/writers/latex.py", line 2252, in unknown_visit
    raise NotImplementedError('Unknown node: ' + node.__class__.__name__)
NotImplementedError: Unknown node: details
```

**Comment by ltalirz** [comment]
looks like this is the pdf build via latex - you can disable it on rtd

**Pull request merged**